Packs/GoogleKubernetesEngine/Playbooks/Google_Kubernetes_Engine_Operations_Generic_Polling.yml

```diff
@@ -146,4 +146,7 @@ inputs:
   required: false
   description: Operation name.
   playbookInputQuery:
-fromversion: 5.5.0
+fromversion: 5.0.0
+tests:
+  -  No tests
```

Comment:
how long is this playbook running? shouldn't it be called in the integration test playbook?